symfony/framework-bundle/7.2/config/packages/framework.yaml

```diff
@@ -2,6 +2,10 @@
 framework:
     secret: '%env(APP_SECRET)%'
 
+    form: { csrf_protection: { token_id: submit } }
+    csrf_protection:
+        double_submit_token_ids: [submit, authenticate, logout]
```

Comment:
I think we need to link to a documentation page here what this means with pros and cons. It's not very self-explaning.